src/pages/docs/tasks/prioritize-tasks.md

```diff
@@ -29,3 +33,11 @@ Within each category, tasks are handled on a first in, first out basis.
 ## Lifecycle phases
 
 Octopus **2024.4** adds a feature to mark a phase within a lifecycle as a priority. See [lifecycle phases with priority](/docs/releases/lifecycles#phases-with-priority) for more information.
+
+## Priority Deployments and Runbooks
+
+:::div{.info}
+Priority runbooks requires version <insert version here>
```

Comment:
Just a reminder that the version number is needed here, too.

If we don't know it yet, this PR should be a draft.

Reply:
The version number has been updated